core/trino-main/src/main/java/io/trino/execution/scheduler/ExponentialGrowthPartitionMemoryEstimator.java

```diff
@@ -27,32 +29,64 @@
 public class ExponentialGrowthPartitionMemoryEstimator
         implements PartitionMemoryEstimator
 {
+    private static final double ESTIMATION_QUANTILE = 0.9;
```

Comment:
Should we make this configurable?

Reply:
May come useful for experimenting. We may probably fix it to sth. I will add gauge for now.